src/vdb_to_numpy/pybind/vdb_pybind.cpp

```diff
@@ -61,7 +60,6 @@ PYBIND11_MODULE(vdb_pybind, m) {
           "(8, 8, 8) grid containing the floating point values of the leaf "
           "node.",
           "grid"_a);
-    m.def("_extract_triangle_mesh", &ExtractTriangleMesh);
```

Comment:
So we will use other methods to construct mesh from tsdf, not vdb-based?

Reply:
same function already exists in vdbfusion package, we don't need the second one here. We can still visualize it as mesh from vdb grid.

Reply:
https://github.com/PRBonn/vdbfusion/blob/2e4374a6b718e49b36409c89ed24aa71eecba77f/src/vdbfusion/pybind/vdbfusion_pybind.cpp#L141